package.json

```diff
@@ -18,12 +18,12 @@
   "dependencies": {
     "eventemitter2": "^6.4.9",
     "p-each-series": "^2.2.0",
-    "puppeteer": "^19.7.0"
+    "puppeteer": "^20.9.0"
   },
   "devDependencies": {
     "difflet": "^1.0.1",
     "grunt": "^1.6.1",
-    "grunt-contrib-connect": "^3.0.0",
+    "grunt-contrib-connect": "^4.0.0",
     "grunt-contrib-internal": "^8.0.2",
```

Comment:
There is the version 9.0.0 to this as well

Reply:
Ack, I created this release at https://github.com/gruntjs/grunt-contrib-internal/pull/57 after drafting this PR. The PR was blocked on 9.0.0 being released first, because grunt-contrib-internal is what manages our CI configuration. This PR raises the Node requirement to Node 16, which thus needs a change to the CI config to drop the Node 14 jobs.